src/modules/attitude_estimator_q/attitude_estimator_q_main.cpp

```diff
@@ -691,12 +699,13 @@ bool AttitudeEstimatorQ::update(float dt)
 	corr += (k % (_accel - _pos_acc).normalized()) * _w_accel;
 
 	// Gyro bias estimation
-	if (_gyro.length() < 1.0f) {
+	if (spinRate < 0.175f) {
```

Comment:
Do we know that this isn't the main reason for the performance improvement. It is a big differences on the magnitude of rotation rate at which we are estimating gyro biases from 1.0  (57 deg/2) to 0.175 (~6 deg/s).